fastestimator/estimator/estimator.py

```diff
@@ -90,7 +91,10 @@ def _sort_traces(self):
         sorted_traces = []
         available_outputs = {
             "num_devices", "mode", "epoch", "train_step", "batch_idx", "batch_size", "batch", "elapsed_time"
-        } | set(self.network.all_losses)
+        } | set().union(*[{item.outputs} if isinstance(item.outputs, str) else set(item.outputs)
+                          for sublist in (list(op.epoch_dict.values()) if isinstance(op, Scheduler) else [op]
+                                          for op in self.network.ops) for item in sublist])
+        # available_outputs starts with all of our default keys plus all of the outputs listed within any network ops
```

Comment:
so all these efforts is because we don't want to use list as member check?

Reply:
lol, no, we don't have that info anywhere in list form either